invenio_communities/config.py

```diff
@@ -20,7 +20,7 @@
     "frontpage": "/communities",
     "search": "/communities/search",
     "new": "/communities/new",
-    "details": "/communities/<pid_value>",
+    "details": "/communities/<pid_value>/records",
```

Comment:
same question as before

Reply:
It was decided that the url for community front/home page to be `communities/<pid_value>` and based on the theme enabled or not it will redirect to the records page for communities that don't have a homepage to  `communities/<pid_value>/records`